vnext/Chakra/ChakraExecutor.cpp

```diff
@@ -250,7 +250,7 @@ void ChakraExecutor::initOnJSVMThread() {
   SystraceSection s("ChakraExecutor.initOnJSVMThread");
 
   bool enableDebugging = m_instanceArgs.EnableDebugging;
-  bool breakOnNextLine = m_instanceArgs.DebuggerBreakOnNextLine;
+  [[maybe_unused]] bool breakOnNextLine = m_instanceArgs.DebuggerBreakOnNextLine;
```

Comment:
>[[maybe_unused]] [](start = 2, length = 16)

why not use if #DEBUG macro?

Reply:
This is a standard way to say that variable maybe unused. It is used below under USE_EDGEMODE_JSRT condition. The #DEBUG will not do a right thing here.

---
In reply to: [370870108](https://github.com/microsoft/react-native-windows/pull/3958#discussion_r370870108) [](ancestors = 370870108)